generate_qfp.py

```diff
@@ -350,7 +350,7 @@ def _uuid(identifier: str) -> str:
         # General info
         lines.append('(librepcb_package {}'.format(uuid_pkg))
         lines.append(' (name "{}")'.format(full_name))
-        lines.append(' (description "{}")'.format(full_description, generator))
+        lines.append(' (description "{}\\n\\nGenerated with {}")'.format(full_description, generator))
```

Comment:
This seems to be an unrelated change...

Reply:
Read the commit message :wink: This fixes a flake8 error that the parameter `generator` wasn't used :slightly_smiling_face:

Reply:
Ah, makes sense!